packages/@aws-cdk/aws-apigatewayv2/lib/http/api.ts

```diff
@@ -16,10 +17,67 @@ export interface IHttpApi extends IResource {
    */
   readonly httpApiId: string;
 
+  /**
+   * A human friendly name for this HTTP API. Note that this is different from `httpApiId`.
+   */
+  readonly httpApiName?: string;
```

Comment:
Since this is only available to CDK initialized http api (not imported), remove this from the interface and declare it only in `HttpApi`.

Reply:
done